core/src/main/res/values/strings.xml

```diff
@@ -166,6 +166,7 @@
     <string name="share_website_url_label">Website address</string>
     <string name="share_feed_url_label">Podcast feed URL</string>
     <string name="feed_delete_confirmation_msg">Please confirm that you want to delete the podcast \"%1$s\" and ALL its episodes (including downloaded episodes).</string>
+    <string name="feed_delete_confirmation_msg_batch">Please confirm that you want to remove the selected podcasts and ALL downloaded episodes except local files.</string>
```

Comment:
I would remove the reference to local files here. Could be confusing to users who never used local podcasts.

```suggestion
    <string name="feed_delete_confirmation_msg_batch">Please confirm that you want to remove the selected podcasts and ALL downloaded episodes.</string>
```